watchdog-config/extra-sql-queries.yml

```diff
@@ -75,3 +75,400 @@ users_with_insificient_disk:
+  metrics:
+    - partner_name:
+        usage: "LABEL"
+    - reported:
+        usage: "LABEL"
+    - due:
+        usage: "GAUGE"
+    - delivered:
+        usage: "LABEL"
+    - scheduled:
+        usage: "LABEL"
+    - gateway_state:
+        usage: "LABEL"
+    - last_delivered:
+        usage: "LABEL"
+    - cht_instance:
+        usage: "LABEL"
+
+invalid_docs:
+  query: |
+    SELECT
+        docs.partner_name,
+        day::varchar(255) AS reported,
+        invalid_doc_count,
+        replace(monitoring_urls.url, 'https://', '') as cht_instance
+    FROM app_monitoring_invalid_docs AS docs
+    LEFT JOIN
+        monitoring_urls ON docs.partner_name=monitoring_urls.partner_name
+  metrics:
+    - partner_name:
+        usage: "LABEL"
+    - reported:
+        usage: "LABEL"
+    - invalid_doc_count:
+        usage: "GAUGE"
+    - cht_instance:
+        usage: "LABEL"
+
+invalid_doc_errors:
+  query: |
+    SELECT
+        docs.partner_name,
+        day::varchar(255) AS reported,
+        code,
+        count,
+        replace(monitoring_urls.url, 'https://', '') as cht_instance
+    FROM app_monitoring_invalid_doc_errors AS docs
+    LEFT JOIN
+        monitoring_urls ON docs.partner_name=monitoring_urls.partner_name
+  metrics:
+    - partner_name:
+        usage: "LABEL"
+    - reported:
+        usage: "LABEL"
+    - code:
+        usage: "LABEL"
+    - count:
+        usage: "GAUGE"
+    - cht_instance:
+        usage: "LABEL"
+
+tasks_by_state:
+  query: |
+    SELECT
+        tasks.partner_name,
+        day::varchar(255) AS reported,
+        task,
+        draft,
+        ready,
+        cancelled,
+        completed,
+        failed,
+        replace(monitoring_urls.url, 'https://', '') as cht_instance
+    FROM app_monitoring_tasks_by_state AS tasks
+    LEFT JOIN
+        monitoring_urls ON tasks.partner_name=monitoring_urls.partner_name
+  metrics:
+    - partner_name:
+        usage: "LABEL"
+    - reported:
+        usage: "LABEL"
+    - task:
+        usage: "LABEL"
+    - draft:
+        usage: "GAUGE"
+    - ready:
+        usage: "GAUGE"
+    - cancelled:
+        usage: "GAUGE"
+    - completed:
+        usage: "GAUGE"
+    - failed:
+        usage: "GAUGE"
+    - cht_instance:
+        usage: "LABEL"
+
+tasks_by_user:
+  query: |
+    SELECT
+        tasks.partner_name,
+        chw,
+        task,
+        docs,
+        cancelled,
+        completed,
+        replace(monitoring_urls.url, 'https://', '') as cht_instance
+    FROM app_monitoring_tasks_by_user AS tasks
+    LEFT JOIN
+        monitoring_urls ON tasks.partner_name=monitoring_urls.partner_name
+  metrics:
+    - partner_name:
+        usage: "LABEL"
+    - chw:
+        usage: "LABEL"
+    - task:
+        usage: "LABEL"
+    - docs:
+        usage: "GAUGE"
+    - cancelled:
+        usage: "GAUGE"
+    - completed:
+        usage: "GAUGE"
+    - cht_instance:
+        usage: "LABEL"
+
+form_completions:
+  query: |
+    SELECT
```

Comment:
same as above - this one returns 18k results which seems a bit verbose

Reply:
Limited to last 90 days

Reply:
please revisit this - it looks like it didn't get added to [this query when I check the most recent version](https://github.com/medic/cht-app-monitoring-data-ingest/blob/34e28d277794ec2607bd0d4cc6178b9ad5a2f5a6/watchdog-config/extra-sql-queries.yml#L405).

Reply:
Updated

Reply:
Walking this back as limitng by time frame will not have a significant impact on the result set size. It has to aggregate by all unique form ID's & users.

Reply:
Will limit time window in the upstream function

Reply:
I defer to you, but I still can't see how MoH Mali CHW, for example, would actually want to look at almost 2.3k rows in anything but a much more summarized way.  If you thinks it's still useful to get this may rows back (18k total) we can keep it!